docs/data/toolpad/core/pages.ts

```diff
@@ -60,86 +60,44 @@ const pages: MuiPage[] = [
     pathname: '/toolpad/core/components-group',
     title: 'Components',
     children: [
-      { pathname: '/toolpad/core/components', title: 'All components' },
+      { pathname: '/toolpad/core/all-components', title: 'All components' },
       {
-        pathname: '/toolpad/core/components/layout-group',
-        subheader: 'Layout',
-        children: [
-          {
-            pathname: '/toolpad/core/components/dashboard-layout',
-            title: 'Dashboard Layout',
-          },
-        ],
-      },
-      {
-        pathname: '/toolpad/core/components/data-group',
-        subheader: 'Data',
+        pathname: '/toolpad/core/provider-group',
+        subheader: 'Provider',
         children: [
           {
-            pathname: '/toolpad/core/components/data-grid',
-            title: 'Data Grid',
-          },
-          {
-            pathname: '/toolpad/core/components/line-chart',
-            title: 'Line Chart',
+            pathname: '/toolpad/core/react-app-provider',
+            title: 'App Provider',
           },
         ],
       },
       {
-        pathname: '/toolpad/core/components/filter-group',
-        subheader: 'Filters',
+        pathname: '/toolpad/core/layout-group',
+        subheader: 'Layout',
         children: [
           {
-            pathname: '/toolpad/core/components/select-filter',
-            title: 'Select',
+            pathname: '/toolpad/core/react-dashboard-layout',
```

Comment:
Not sure if we should follow this `react-...` naming pattern for pages, I saw all the other MUI libraries are doing it? But we can do it differently too.